.github/workflows/proof_ci.yaml

```diff
@@ -114,6 +130,36 @@ jobs:
           sudo apt-get update
           sudo apt-get install --no-install-recommends --yes ./litani.deb
           rm ./litani.deb
+      - name: Install Z3 ${{ env.Z3_VERSION }}
+        if: ${{ env.Z3_VERSION != 'latest' }}
+        shell: bash
+        run: |
+          curl -o z3.zip -L \
+            https://github.com/Z3Prover/z3/releases/download/z3-${{ env.Z3_VERSION }}/z3-${{ env.Z3_VERSION }}-x64-glibc-2.31.zip
+          sudo apt-get install --no-install-recommends --yes unzip
+          unzip z3.zip
+          cd z3-${{ env.Z3_VERSION }}-x64-glibc-2.31/bin; echo "$(pwd)" >> $GITHUB_PATH
+          cd z3-${{ env.Z3_VERSION }}-x64-glibc-2.31/bin; ./z3 --version
+          rm ./z3.zip
+      - name: Build and Install Bitwuzla ${{ env.BITWUZLA_VERSION }}
+        if: ${{ env.BITWUZLA_VERSION != 'latest' }}
```

Comment:
Same here: does this actually make sense?

Reply:
Same again - in future, we might support a "latest" build of Bitwuzla, so I decided to leave the test there.